### ISSUE TITLE
[console] Set defaults for commands on user config file.

### ISSUE THREAD

**Issue by jmolivas** [hechoendrupal/DrupalConsole#515]
I am planning to set defaults for commands on that config file at home directory `~/.console/config.yml`, will probably look something like this:

```
command:
  generate:
    module:
      options: 
        module: My example module
        machine-name: example
    controller:
      options: 
        module: example
        class-name: ExampleController
```

The idea is to take advantage of user config file at `~/.console/config.yml` file on user home directory. 

Feature based on this issue at d.o. https://www.drupal.org/node/2392831

**Comment by omero** [comment]
It's very interesting this enhancement ... comes to me, the idea to have different default files and calling when it needed like `alias` to create for example modules with a default package :smiley:

**Comment by jmolivas** [comment]
you mean like pre-sets of data, I have not think on that possibility but could make  sense

**Comment by jmolivas** [comment]
That what you mentioned @omero maybe fits better on the issue https://github.com/hechoendrupal/DrupalAppConsole/issues/516

Using the second option 
Reading from a YAML manifest file. 
$ drupal chain:commands --manifest=PATH_TO_MANIFEST_FILE.yml

**Comment by enzolutions** [comment]
The default values could be generated using the command --generate-chain that will be included in next release  0.7.15

**Comment by jmolivas** [comment]
The format for setting the defaults and the chain are not the same but we can add a global option --default and output the generated code to be added on the `config.yml` file
